event_manager:  Change logging in Event Manager (#6811)
nrfconnect/sdk-nrf: doc/nrf/libraries/others/event_manager.rst

.. note::
	There is a deprecated way of logging Event Manager events by writing a string to the provided buffer that will be supported until a future release of |NCS|.

> 	To use it you need to set :kconfig:`CONFIG_EVENT_MANAGER_USE_DEPRECATED_LOG_FUN` Kconfig option.

Reviewer: ```suggestion
	To use the deprecated way, you need to set the :kconfig:`CONFIG_EVENT_MANAGER_USE_DEPRECATED_LOG_FUN` option.
```

> 	You can then use both ways of logging and event manager will figure out which way is used based on the type of the logging function passed.

Reviewer: ```suggestion
	You can then use both ways of logging events.
	Event manager figures out which way to be used based on the type of the logging function passed.
```
